TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/console_link/workflow/commands/configure.py

```diff
@@ -118,16 +164,25 @@ def _save_config(store, new_config: WorkflowConfig, session_name: str):
         raise click.ClickException(f"Failed to save configuration: {e}")
 
 
-def _handle_stdin_edit(store, session_name: str):
+def _handle_stdin_edit(wf_config_store, secret_store, session_name: str):
     """Handle configuration edit from stdin"""
     stdin_stream = click.get_text_stream('stdin')
     stdin_content = stdin_stream.read()
 
     new_config = _parse_config_from_stdin(stdin_content)
-    _save_config(store, new_config, session_name)
+    _save_config(wf_config_store, new_config, session_name)
+
+    existing_secrets, missing_secrets = _get_basic_creds_secrets_in_config(secret_store, new_config)
+    if existing_secrets:
+        click.echo(f"Found {len(existing_secrets)} existing secrets that will be used for HTTP-Basic authentication "
```

Comment:
I think that in the non-interactive case, we should be using logging so it's consistent with all our other code and we can control verbosity, etc.